plugins/pom.xml

```diff
@@ -31,5 +31,32 @@
          <artifactId>jboss-logging</artifactId>
          <version>3.3.0.Final</version>
       </dependency>
+
+      <!-- needed to avoid receive NoClassDefFoundError: com/arjuna/ats/arjuna/common/arjPropertyManager
```

Comment:
What is the plugin that requires this dependency? Could you share the stack trace ?

Reply:
```
  [echo] Packaging plugin infinispan90...
     [copy] Copying 90 files to /Users/gustavo/git/radargun/target/distribution/RadarGun-3.0.0-SNAPSHOT/plugins/infinispan90
     [java] Exception in thread "main" java.lang.NoClassDefFoundError: com/arjuna/ats/arjuna/common/arjPropertyManager
     [java] 	at org.radargun.service.InfinispanEmbeddedService.<clinit>(InfinispanEmbeddedService.java:41)
     [java] 	at java.lang.Class.forName0(Native Method)
     [java] 	at java.lang.Class.forName(Class.java:264)
     [java] 	at org.radargun.ServiceHelper.loadServices(ServiceHelper.java:104)
     [java] 	at org.radargun.config.PluginSchemaGenerator.main(PluginSchemaGenerator.java:61)
     [java] Caused by: java.lang.ClassNotFoundException: com.arjuna.ats.arjuna.common.arjPropertyManager
     [java] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
     [java] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
     [java] 	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
     [java] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```